### ISSUE TITLE
[Automate][Chests Anywhere] Add a 'Stack Only' option for Taking and Putting

### ISSUE THREAD

**Issue by Rahmiel** [Pathoschild/LookupAnything#823]
Not an issue but an enhancement, though I am not well versed in the ways of modding. I was wondering if an option could be added to chests that forces Automate to do the following.

Prioritize the chest if it already has the item
Ignore the chest if it doesn't

This would allow for better routing IMHO, and would allow for creating chests of last resort more easily.

**Comment by cosmicdreams** [comment]
I came here to ask the exact same thing.  What a great description, no notes.

But I wish to contribute something.  So here's a use case:

I created a Shed I call "Factory" and it in I want to make simple machines.  I create Chest that is responsible for taking all incoming crops that I want to either preserve, turn into seeds or turn into wine / juice.  With this feature addition I can have control over the flow if I add in chests that have 1 item for each item that should be sent to each machine.

Let's say I don't want to send crops to the Keg if they aren't going to be turned into wine.  I create a combination of machines with a chest that has at least 1 item for each crop that can be turned into wine.  Then I add a hopper (while using the super hopper mod) that forces the item into another chest that will feed the Kegs.  If the crop doesn't pass this filter it gets sent to the Preserves jar.  If the Preserves jar is busy, it is turned into seeds.

**Technical implementation notes:**
In the end I think what we're talking about here is to have the ability to have a chest create an inclusion list.  If the ID for an item isn't in the list, then Automate doesn't put the item into the chest.  The novel idea with this feature request is have the chest itself maintain that inclusion list by generating an array of ids from the items that are in it.
